src/Integration/Binding/BindingController.cs

```diff
@@ -158,24 +158,20 @@ void IBindingWorkflowExecutor.BindProject(BindCommandArgs bindingArgs)
 
             var solutionBindingOp = new SolutionBindingOperation(
                 host,
-                bindingArgs.Connection,
-                bindingArgs.ProjectKey,
-                bindingArgs.ProjectName,
                 modeToBind,
                 host.Logger);
 
             var unboundProjectFinder = new UnboundProjectFinder(host, host.Logger);
 
-            var cSharpVBBindingConfigProvider = new CSharpVBBindingConfigProvider(host.SonarQubeService, nugetBindingOp, currentConfiguration, host.Logger);
```

Comment:
This was an oversight on my part -- all the calculation should be done based on _new configuration_, and not on the current one.